### PR TITLE
chore(replays): Rename replay_click prefix to click

### PR DESCRIPTION
**Pull request by cmanallen**

### CODE REVIEW

**Review by JoshFerge — APPROVED** [review]
click clack

**Review by billyvg — APPROVED** [review]

**Comment by billyvg** [comment]
Frontend update here: https://github.com/getsentry/sentry/pull/47184

File: src/sentry/replays/blueprints/api.md [hunk 1]
```diff
@@ -53,18 +53,18 @@ This document is structured by resource with each resource having actions that c
...
+    | click.alt         | string        | The alt attribute of the HTML element.                         |
+    | click.class       | array[string] | An array of HTML element classes.                              |
+    | click.id          | string        | The ID of an HTML element.                                     |
+    | click.label       | string        | The aria-label attribute of an HTML element.                   |
+    | click.role        | string        | The role of an HTML element.                                   |
+    | click.tag         | string        | Valid HTML5 tag name.                                          |
+    | click.testid      | string        | The data-testid of an HTML element. (omitted from public docs) |
+    | click.textContent | string        | The text-content of an HTML element.                           |
+    | click.title       | string        | The title attribute of an HTML element.                        |
+    | click.selector    | string        | A valid CSS selector.                                          |
```
Comment (ryan953) [inline src/sentry/replays/blueprints/api.md:67]: sorted

```suggestion
    | click.selector    | string        | A valid CSS selector.                                          |
    | click.tag         | string        | Valid HTML5 tag name.                                          |
    | click.testid      | string        | The data-testid of an HTML element. (omitted from public docs) |
    | click.textContent | string        | The text-content of an HTML element.                           |
    | click.title       | string        | The title attribute of an HTML element.                        |
```

File: src/sentry/replays/query.py [hunk 2]
```diff
@@ -434,18 +434,16 @@ class ReplayQueryConfig(QueryConfig):
...
+    click_alt = ListField(field_alias="click.alt", is_sortable=False)
+    click_class = ListField(field_alias="click.class", query_alias="clickClass", is_sortable=False)
+    click_id = ListField(field_alias="click.id", is_sortable=False)
+    click_aria_label = ListField(field_alias="click.label", is_sortable=False)
+    click_role = ListField(field_alias="click.role", is_sortable=False)
+    click_tag = ListField(field_alias="click.tag", is_sortable=False)
+    click_testid = ListField(field_alias="click.testid", is_sortable=False)
+    click_text = ListField(field_alias="click.textContent", is_sortable=False)
+    click_title = ListField(field_alias="click.title", is_sortable=False)
+    click_selector = Selector(field_alias="click.selector", is_sortable=False)
```
Comment (ryan953) [inline src/sentry/replays/query.py:446]:
```suggestion
    click_selector = Selector(field_alias="click.selector", is_sortable=False)
    click_tag = ListField(field_alias="click.tag", is_sortable=False)
    click_testid = ListField(field_alias="click.testid", is_sortable=False)
    click_text = ListField(field_alias="click.textContent", is_sortable=False)
    click_title = ListField(field_alias="click.title", is_sortable=False)
```

**Review by ryan953 — APPROVED** [review]

**Pull request merged**